src/convertToObject.js

```diff
@@ -6,7 +6,16 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const letters = sourceString.split(';');
+  const result = letters.reduce((accumulator, value, index) => {
+    const params = value.split(':').map((element) => element.trim());
+
+    accumulator[params[0]] = params[1];
```

Comment:
You can use destruction for `params`
```suggestion
    const params = value.split(':').map((element) => element.trim());
    
    const [key, value] = params;

    accumulator[key] = value;
```